boards/arm/olimex_stm32_p103/olimex_stm32_p103_defconfig

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32F1X=y
+CONFIG_SOC_STM32F103XB=y
+# 64MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=64000000
```

Comment:
This MCU runs on 72 MHz by default. Is there a reason to specify only 64 MHz for this board?

Please see also `nucleo_f103rb` board.